### DISCUSSION TITLE
Add gear preview to transmogging

### DISCUSSION THREAD

**Discussion by Shredon1** [chromiecraft/chromiecraft#5768] [Suggestions]
Would make picking transmog items 10x easier

**Comment by Vyperhand** [comment]
As discussed in discord, would require changing the NPC from a Gossip interface to a Vendor or custom interface.    I would be willing to put a bounty on this one if it were possible.

**Comment by Helias** [comment]
Now the transmog NPC is a vendor, so I think we can close this discussion, thanks for the suggestion!
